### ISSUE TITLE
Problem on fintuning in newly collected dataset

### ISSUE THREAD

**Issue by xsw1208** [openvla/openvla#32]
Thank you for your wonderful open-sourced VLA models. For fintuning stage, I still have some questions to be solved.
I used RLBench data collected by myself to prepare RLDS data following your repository [rlds_dataset_builder](https://github.com/kpertsch/rlds_dataset_builder). I find there is something missing in guidelines to train on a new dataset.

For training stage:
I need also register my dataset in prismatic/vla/datasets/rlds/oxe/transforms.py and prismatic/vla/datasets/rlds/oxe/configs.py.

After training:
unnorm key for my dataset seems not automatically added in config.json, and I need to copy it from dataset_statics.json

For evaluation:
After I did things above, the action output seems fit in required action space. But even though I got action accuracy higher than 95% in training stage, the visualized actions can not even complete tasks from the training set. Sometimes it moves to the target, but it will then move back.

I want to find the reason why I got bad result in evaluation stage:
1. Is the fix I did for training stage and after training stage correct? Have you already provided interface for convinient use of new dataset.
2. I use 128 * 128 RGB image as observation input, which is recommanded in  [rlds_dataset_builder](https://github.com/kpertsch/rlds_dataset_builder), but I see your vision encoder uses 224 * 224 to resize. Do I also need higher resolution image as input?
3. I set image_aug to be False,  how will it influece accuracy?
4. I used wrist image to train and test. What viewpoint do you recommand to train and eval?
5. I used simulation environment, is there big real-to-sim gap? How will it success on simulation environment?

Thank you!

**Comment by kpertsch** [comment]
Hi! Thanks for your questions.
Indeed, you need to register new datasets in the files you mentioned -- we should document this better!
Re unnorm_key: this is an unresolved issue -- the HF automodel does not have the new datasets_statistics in the finetuning stage, so indeed I have loaded them manually from the file for finetuned models so far.

Regarding your performance questions:
- I would recommend storing images at a higher resolution (eg 256) and then let our model downsize them to 224, rather than making them smaller to start with
- it is really important that your inference pipeline mirrors exactly all preprocessing steps you did for going from raw RLBench data to model inputs. this includes any resizes (make sure to also match the resize method) -- esp for sim data with very little variation, small artifacts of resizing can really throw policies off
- re camera view: I would suggest you start with a third-person view and see whether the model behaves more reasonably -- OpenVLA was only trained with 3rd person views, and the distribution shift to a wrist view would be quite large.
- we usually use image_aug for real world finetuning, but in sim it may not be that important since the eval scene camera viewpoint / lighting match training exactly
